cms/templates/js/course-outline.underscore

```diff
@@ -246,6 +246,48 @@ if (is_proctored_exam) {
                         <% } %>
                     </p>
                 </div>
+                <% if (course.get('self_paced') && course.get('is_custom_pls_active') && xblockInfo.get('due_num_weeks')) { %>
+                    <div class="status-grading">
+                        <p>
+                            <span class="icon fa fa-calendar" aria-hidden="true"></span>
+                            <span class="status-custom-grading-date">
+                                <%- edx.StringUtils.interpolate(
```

Comment:
Want to give me a quick tldr on `edx.StringUtils.interpolate`?

Reply:
yeah so it basically does what you mentioned here https://github.com/edx/edx-platform/pull/28175#discussion_r670581494 where the string created is interpolated by the parameters, which in this case is the number of weeks. What you mentioned in the comment didn't work so this in an alternative working solution